examples/halo_test/halo_test.f90

```diff
@@ -86,7 +86,7 @@ subroutine initialise()
 
     call random_seed(size = n)
     allocate(seed(n))
-    seed = nrank
+    seed = nrank+1
```

Comment:
Why this change? Just thinking it could change people's results (slightly)

Reply:
This is for NVHPC that does not like seed starting from 0. I modified this also in Xcompact3d a while ago. In terms of results should be ok since compute the divergence of an initial field generated by rand. The initial field is the same for both methods just computed in two different ways (transpose and halo) therefore the output must be the same.

Reply:
ah - I thought this was being set elsewhere, not in the test case!